src/lib/circuit_value.ts

```diff
@@ -271,12 +272,15 @@ function circuitMain(
   }
 
   target.snarkyMain = (w: Array<any>, pub: Array<any>) => {
-    let args = [];
-    for (let i = 0; i < numArgs; ++i) {
-      args.push((publicIndexSet.has(i) ? pub : w).shift());
-    }
+    let [, result] = withContext({ inCheckedComputaton: true }, () => {
```

Comment:
Nit: Spelling of `inCheckedComputaton` (and other references too it)

Reply:
good catch!